index.html

```diff
@@ -103,5 +103,6 @@ <h1>Tasks</h1>
         </svg>
       </button>
     </main>
+    <script src="index.js" defer="true"></script>
```

Comment:
No value required (following MDN)

```suggestion
    <script src="index.js" defer></script>
```

Reply:
Thank you!